stock_orderpoint_manual_procurement/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from datetime import datetime
+from odoo.addons import decimal_precision as dp
+from odoo.tools import float_compare, float_round
+
+
+UNIT = dp.get_precision('Product Unit of Measure')
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = 'stock.warehouse.orderpoint'
+
+    procure_recommended_qty = fields.Float(
```

Comment:
@lreficent Maybe this could be externalized in a more generic module with values that help users to manage orderpoints (such as virtual_available quantity or ...) ?

But that is out of scope of migration process.

Reply:
This is also interesting, the need for this field and not for the manual procurement mechanism could exist. I'll look into this too

Reply:
Hi @rousseldenis 

Rethinking about this, I wonder what would be the use of these field by themselves, I mean: What would be the point of having `procure_recommended_qty` visible if you cannot procure manually?

Reply:
@lreficent Yes, that's ok. I was thinking even if user should see the next reordering quantities that will be ordered (in any case).

Let behaviour like this